cmake/IRODSConfig.cmake.not_yet_installed.in

```diff
@@ -81,6 +81,9 @@ endif()
 if(NOT DEFINED IRODS_PLUGINS_DIRECTORY)
 	set(IRODS_PLUGINS_DIRECTORY "@IRODS_PLUGINS_DIRECTORY@")
 endif()
+if(NOT DEFINED IRODS_CLIENTS_DIRECTORY)
+	set(IRODS_PLUGINS_DIRECTORY "@IRODS_PLUGINS_DIRECTORY@")
```

Comment:
is this ... correct?    should it be IRODS_CLIENTS_DIRECTORY twice?

Reply:
Thank you